config/manager/usde-config.yaml

```diff
@@ -78,13 +75,43 @@ data:
       - path: spec.redis.external.sentinelUrl
       - path: spec.redis.external.masterName
     recreate:
-      # TODO: add the recreate fields
-    progressive:
-      # TODO: add the progressive fields
+      # Changing persistence settings and number of replicas needs to recreate both the ISBSvc and also the pipelines associated to it to 
+      # recreate the buckets and buffers which are only recreated when a pipeline is started.
+      - path: spec.jetstream.replicas
+      - path: spec.redis.native.replicas
+      - path: spec.jetstream.persistence
+        includeSubfields: true
+      - path: spec.redis.native.persistence
+        includeSubfields: true
```

Comment:
nit: if we're going to include redis fields in here, we may as well also include the changing from redis to jetstream and jetstream to redis (even though Derek said it's "not allowed", that was only because redis doesn't have all of the features)